frontend/next.config.js

```diff
@@ -18,6 +18,12 @@ const {
 } = process.env;
 
 module.exports = withSourceMaps({
+  i18n: {
+    locales: ["default", "en", "it"],
+    defaultLocale: "default",
+    localeDetection: false,
```

Comment:
just curious, why is this false?

Reply:
it was suggested in the nextjs section: https://nextjs.org/docs/advanced-features/i18n-routing#prefixing-the-default-locale we can try enabling it and see if it has any issue. I have a few ideas on why it might be required but I need to investigate before saying anything